plugins/org.jkiss.dbeaver.data.transfer.ui/src/org/jkiss/dbeaver/tools/transfer/ui/pages/database/DatabaseConsumerPageMapping.java

```diff
@@ -893,8 +893,19 @@ protected Object doGetValue() {
 
             @Override
             protected void doSetValue(Object value) {
-                if (combo != null && !combo.isDisposed()) {
-                    combo.setText(CommonUtils.toString(value));
+                if (combo == null || combo.isDisposed()) {
+                    return;
+                }
+                if (value == null) {
+                    combo.setText("");
+                } else {
+                    if (value instanceof DBPNamedObject dbpNamedObject) {
```

Comment:
This is something a label provider should do. Is the value not constrained to a particular type? Can it really be either a `DBPNamedObject`, an `Enum`, or something else?